examples/panel/book-recommender/app.py

```diff
@@ -0,0 +1,132 @@
+"""
+Demonstrates how to use the `ChatInterface` and a `callback` function to respond.
+
+The chatbot Assistant echoes back the message entered by the User.
+
+https://github.com/holoviz-topics/panel-chat-examples/blob/main/docs/examples/basics/basic_chat.py
+"""
+
+import panel as pn
+from openai import OpenAI
+from scipy.spatial import KDTree
+import numpy as np
+
+from rag_book_recommender import get_book_description_by_title, EmbeddingsStore, get_authors, get_embeddings
+
+
+client = OpenAI()
+
+pn.extension()
+
+store = EmbeddingsStore()
+all_authors = get_authors()
```

Comment:
this naming is confusing. I believe  get_authors returns an author -> book titles mapping, right? `all_authors` is innacurate

Reply:
[fixed](https://github.com/ploomber/doc/pull/127/files#diff-99d50d468b15d576674758b1fbde220b390f380dc2398dd8b4024a04218dd2ccR31-R32)